packages/sku/scripts/storybook.js

```diff
@@ -1,21 +1,18 @@
-const path = require('path');
 const { argv } = require('../config/args');
 const gracefulSpawn = require('../lib/gracefulSpawn');
 const { storybookPort } = require('../context');
 const startStorybookPath = require.resolve('@storybook/cli/bin/index');
-const configDir = path.resolve(__dirname, '../config/storybook/start');
 const { watchVocabCompile } = require('../lib/runVocab');
-const { setUpStorybookPreviewFile } = require('../lib/storybook');
+const { setUpStorybookConfigDirectory } = require('../lib/storybook');
 
 // Unshift args to allow pushing --ci as an arg during storybook-config tests
 argv.unshift('--quiet');
-argv.unshift('--config-dir', configDir);
```

Comment:
With the removal of this and reliance on the `.storybook` directory, should we validate the availability of this? Not sure what the DX is like if this goes wrong, or in a monorepo or some situation with the folder isnt available to cwd.

Reply:
Looking at [the source code](https://github.com/storybookjs/storybook/blob/a810931c9321c0d95d2eb48e23f62856f11b5592/code/lib/core-common/src/utils/get-storybook-info.ts#L86-L108), it looks like it's doing a relative path `fs.existsSync` to check for the config files, so it won't pick up `.storybook` in a parent folder for example. However, this isn't any different to how it worked before.

[The error on storybook's side](https://github.com/storybookjs/storybook/blob/a810931c9321c0d95d2eb48e23f62856f11b5592/code/lib/cli/src/add.ts#L66-L69) could be a bit more specific though.

Reply:
Maybe we consider improving the DX down the track if this scenario eventuates then. Im happy with that.